### PR TITLE
Fix sanity check of netcdf4-python

### PR DESCRIPTION
**Pull request by wpoely86**
Now works for both egg and non-egg versions

Needed for hpcugent/easybuild-easyconfigs#2221

### CODE REVIEW

File: easybuild/easyblocks/n/netcdf4_python.py [hunk 1]
```diff
@@ -70,11 +69,8 @@ def test_step(self):
 
     def sanity_check_step(self):
         """Custom sanity check for netcdf4-python"""
-        shlib_ext = get_shared_lib_ext()
-
         custom_paths = {
-            'files': ['bin/nc3tonc4', 'bin/nc4tonc3', 'bin/ncinfo'] +
-                     [os.path.join(self.pylibdir, x) for x in ['netCDF4.%s' % shlib_ext, 'netCDF4_utils.py', 'netcdftime.py']],
```
Comment (boegel) [inline easybuild/easyblocks/n/netcdf4_python.py]: is the `.so` also in the `.egg`? if not, keep it separate as a file to check on

Comment (wpoely86) [inline easybuild/easyblocks/n/netcdf4_python.py]: it's all in the egg

Comment (boegel) [inline easybuild/easyblocks/n/netcdf4_python.py]: OK, then this is good enough I guess, especially since it'll do the `import` check too.

**Comment by boegel** [comment]
Thanks @wpoely86!

**Pull request merged**